.github/workflows/lint.yml

```diff
@@ -2,25 +2,22 @@ name: lint
 
 on:
   push:
-    branches: [main, master]
+    branches: [main]
```

Comment:
This seems unrelated to xtask?

Reply:
I've now moved this to a seperate commit which cleans up `lint.yml` and switches to a different rust toolchain manager